TFT/src/User/Menu/StatusScreen.c

```diff
@@ -290,8 +300,8 @@ void menuStatus(void)
       drawStatusScreenMsg();
 
     scrollMsg();
-    key_num = menuKeyGetValue();
 
```

Comment:
Please revert this change. I prefer it the way it was.

Reply:
ok, no problem to revert those although they have been made just to be compliant to other files (Printing menu etc...) not reviewed by me